yt_dlp/extractor/eplus.py

```diff
@@ -1,15 +1,24 @@
+import json
+import random
+
 from .common import InfoExtractor
 from ..utils import (
     ExtractorError,
     try_call,
     unified_timestamp,
+    urlencode_postdata,
+    xpath_text,
 )
 
 
-class EplusIbIE(InfoExtractor):
-    IE_NAME = 'eplus:inbound'
-    IE_DESC = 'e+ (イープラス) overseas'
-    _VALID_URL = r'https?://live\.eplus\.jp/ex/player\?ib=(?P<id>(?:\w|%2B|%2F){86}%3D%3D)'
+class EplusIE(InfoExtractor):
```

Comment:
Renaming the class creates issues with archives. Please revert. `IE_NAME` is all that user will see, so you can just change that.

Reply:
@pukkandan , sorry, I reverted it at 061a17f18b293832fe2b4b33372d437b17ecabc6 .

I think the "Ib" may confuse developers in the future, maybe I should make "EplusBaseIE", "EplusIbIE" and "EplusJpIE"?

Reply:
imo, it's fine as is